sdk/storage/azure-storage-file-datalake/src/main/java/module-info.java

```diff
@@ -13,12 +13,9 @@
     exports com.azure.storage.file.datalake.implementation.util to
         com.fasterxml.jackson.databind;
 
-    exports com.azure.storage.file.datalake.implementation.models to
-        com.fasterxml.jackson.databind,
-        com.azure.core;
-
     opens com.azure.storage.file.datalake.implementation.models to
         com.fasterxml.jackson.databind,
         com.azure.core;
+    exports com.azure.storage.file.datalake.implementation.models;
```

Comment:
We shouldn't be exporting implementation without specific targets, did this PR run into a roadblock with modules that caused this? If it was a testing issue we should add an `--adds-export com.azure.storage.file.datalake/com.azure.storage.file.datalake.implementation.models=<the module that needs this or ALL-UNNAMED` to the `javaModulesSurefireArgLine` configuration in the POM